### PR TITLE
Last changes

### PR DESCRIPTION
**Pull request by DimosTzanakos**
Code changes.
Pictures at the right folders.
Added illustrated documentation.

### CODE REVIEW

**Comment by namezys** [comment]
you have not append .gradle files

File: build/reports/tests/test/classes/elte_tsp_2019s_htel.AppTest.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```
Comment (namezys) [inline build/reports/tests/test/classes/elte_tsp_2019s_htel.AppTest.html]: build again

**Comment by DimosTzanakos** [comment]
have you checked my last commit in this branch?
I think i have added gradle files.

**Comment by DimosTzanakos** [comment]
What about now?

**Comment by namezys** [comment]
you have to remove build, .gradle, etc

**Comment by DimosTzanakos** [comment]
ok. So the build folder and the .gradle must not be in the repo?

**Comment by DimosTzanakos** [comment]
Niko, the pictures for the rooms must be in build/classes/java/main.
Thats where gradle looks for the pictures.
If not, the code for the ViewRoomsFrame does not work.

**Pull request closed without merging**